pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -163,9 +168,9 @@ func getIgnitionConfig(assetsDir string, profile *performancev1alpha1.Performanc
 		})
 	}
 
-	nonIsolatedCpus := profile.Spec.CPU.NonIsolated
+	ReservedCpus := profile.Spec.CPU.Reserved
```

Comment:
ReservedCpus -> reservedCpus
true for all occurences

Reply:
I think I fixed this one for David.

Reply:
yup, it's fixed, thanks @MarSik